### PR TITLE
Update node

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Upgrade Node version due to critical CVE: https://nodejs.org/en/blog/vulnerability/february-2020-security-releases/

### CODE REVIEW

**Comment by Aschen** [comment]
@alexandrebouthinon Do you know what is the dockercloud CI? 
I can't access it :(

File: core-dev-aarch64/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM resin/aarch64-debian:stretch
 LABEL io.kuzzle.vendor="Kuzzle <support@kuzzle.io>"
 LABEL description="Core development stack"
 
-ENV NODE_12_VERSION=12.13.0
+ENV NODE_12_VERSION=12.15.0
```
Comment (alexandrebouthinon) [inline core-dev-aarch64/Dockerfile]:
```suggestion
ENV NODE_12_VERSION=12.16.0
```

File: core-dev-armhf/Dockerfile [hunk 2]
```diff
@@ -3,7 +3,7 @@ FROM resin/armv7hf-debian:stretch
 LABEL io.kuzzle.vendor="Kuzzle <support@kuzzle.io>"
 LABEL description="Core development stack"
 
-ENV NODE_12_VERSION=12.13.0
+ENV NODE_12_VERSION=12.15.0
```
Comment (alexandrebouthinon) [inline core-dev-armhf/Dockerfile]:
```suggestion
ENV NODE_12_VERSION=12.16.0
```

File: core-dev/Dockerfile [hunk 3]
```diff
@@ -5,7 +5,7 @@ LABEL description="Enhance the Kuzzle core with ease"
 
 WORKDIR /var/app
 ENV TERM=xterm
-ENV NODE_12_VERSION=12.13.0
+ENV NODE_12_VERSION=12.15.0
```
Comment (alexandrebouthinon) [inline core-dev/Dockerfile]:
```suggestion
ENV NODE_12_VERSION=12.16.0
```

**Pull request merged**